webapp/src/pages/ChatPage/ChatPage.tsx

```diff
@@ -42,23 +42,30 @@ export default function ChatPage() {
     [],
   );
   const navigate = useNavigate();
-
+  const windowIsBig = useMediaQuery(768);
+  const chatButtons = [
+    {
+      buttonSize: windowIsBig ? ButtonSize.LARGE : ButtonSize.SMALL,
```

Comment:
I think this responsibility should be button's one, I mean, handling its size. It should know wheter it should display one version or another depending on screenSize and could be done in pure CSS.

We could have another button component wich inherits from a standard button which adds this media-query behaviour (`ReactiveButton`, or any other better name 😄 ) This way we wouldn't even need to handle queries from `tsx` code

Reply:
You can have two buttons, overlapping, and display one or the other (setting the non relevant one to display:none) depending on the media query. On the other hand, ~what is the principle you would quote to explain that it's not~ why do you think it is not acceptable to write inline styles?

Reply:
done